rust/sdk-processor/src/steps/common/latest_processed_version_tracker.rs

```diff
@@ -12,9 +11,8 @@ use async_trait::async_trait;
 use diesel::{upsert::excluded, ExpressionMethods};
 use processor::{db::common::models::processor_status::ProcessorStatus, schema::processor_status};
 use std::marker::PhantomData;
-use tracing::info;
 
-const UPDATE_PROCESSOR_STATUS_SECS: u64 = 1;
+pub const UPDATE_PROCESSOR_STATUS_SECS: u64 = 1;
```

Comment:
should this be a config thing?

Reply:
Yeah this can be the default, but it should be configurable per processor.

Reply:
will do